nautobot/core/templates/inc/relationships_table_rows.html

```diff
@@ -13,10 +13,19 @@
                     <td>
                         <a href="{% url 'extras:relationshipassociation_list' %}?relationship={{relationship.slug}}&{{side}}_id={{object.id}}">
                             {{ value.queryset.count }}
-                            {% if value.queryset.count > 1 %}
-                                {{ value.peer_type.model_class|meta:"verbose_name_plural" }}
+                            {% comment %}
+                            If peer_type is a plugin model that has subsequently been uninstalled,
+                            peer_type.model_class will be None and we can't access its Meta.
+                            https://github.com/nautobot/nautobot/issues/2077
+                            {% endcomment %}
+                            {% if value.peer_type.model_class is not None %}
+                                {% if value.queryset.count > 1 %}
```

Comment:
While we're in here, I think a `queryset.exists` would be better here since `count` can result in a table scan.
```suggestion
                                {% if value.queryset.exists %}
```

Reply:
No, we need to distinguish between `count > 1` and `count <= 1` to handle pluralization properly, I think? Plus we're already rendering the count itself at line 15 above.

Reply:
Yeah ok got it. I don't think that count is cached, though. It's whatever. It it's a problem in the future we can address it then.